packages/components/rollup.config.js

```diff
@@ -67,7 +67,10 @@ export default {
     "@use-it/event-listener",
     "@jobber/design",
     "@jobber/formatters",
-    "@jobber/hooks",
+    new RegExp("@jobber/hooks/.*"),
+    "zxcvbn",
+    "use-resize-observer/polyfilled",
+    "@apollo/client",
```

Comment:
Just a question for my understanding: Why do these new packages need to get added here now?

Reply:
Because we're now referencing `packages/hooks` [directly](https://github.com/GetJobber/atlantis/pull/1193/files#diff-28205e17d9bc238e98aa2b8ccb53cd68a34ca8b75b40aaa8530997411d44f403) in `packages/components`, we rollup needs to be aware of the external dependencies of `hooks` too.